samples/snippets/test_create_assessment.py

```diff
@@ -71,18 +72,32 @@ def recaptcha_site_key() -> str:
     )
 
 
+@pytest.mark.dependency()
```

Comment:
I think this additional dependency and global `ASSESSMENT_NAME` can be avoided with an `assessment` pytest fixture. The code to create the assessment can move into the fixture, and `test_create_assessment` and `test_annotate_assessment` can both use the fixture.

Reply:
Modified in the update. PTAL.